kubernetes/kube-state-metrics-deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: kube-state-metrics
       containers:
       - name: kube-state-metrics
-        image: quay.io/coreos/kube-state-metrics:v1.3.1
+        image: quay.io/coreos/kube-state-metrics:v1.4.0
```

Comment:
bump addon-resizer image version at the same time?

Reply:
Another PR should be introduced to do this.